scripts/gui/microbe_hud.js

```diff
@@ -105,10 +105,10 @@ function runMicrobeHUDSetup(){
             compoundHydrogenSulfide: randomBetween(0, hydrogenSulfide),
             HydrogenSulfideMax: hydrogenSulfide,
         });
-		
-		// Pseudo population code
-		document.getElementById("populationCount").textContent = 27;
-		
+
+        // Pseudo population code
+        updatePopulation(Math.floor(Math.random() * 50) + 0);
```

Comment:
Using `randomBetween` would be consistent with the other GUI preview value generation.